sphinxext/altairgallery.py

```diff
@@ -355,7 +355,7 @@ def main(app):
             example["prev_ref"] = "gallery_{name}".format(**prev_ex)
         if next_ex:
             example["next_ref"] = "gallery_{name}".format(**next_ex)
-        Path(target_dir, f"{example["name"]}.rst").write_text(
+        Path(target_dir, example["name"] + ".rst").write_text(
```

Comment:
Thinking this would've been possible if [docbuild](https://github.com/vega/altair/blob/18a2c3c237014591d172284560546a2f0ac1a883/.github/workflows/docbuild.yml#L13) was using the `python-version: "3.10"`.

Would there be a specific reason for using `3.10` here?
I'd understand `3.8` or `3.12`, but seems unlikely there's a dependency on `3.10`.

Reply:
Yeah we just have to use a version but it can be any of the supported ones by Altair, I'm also not aware of any specific dependency on 3.10 for the documentation. Feel free to bump it to 3.12 and if `docbuild` still runs through that should be fine.

Reply:
Fixed in [`e935ad9` (#3431)](https://github.com/vega/altair/pull/3431/commits/e935ad9a6553828d7bd58bc0b5a6021afa579858)